test/src/sdk-harness/README.md

```diff
@@ -9,11 +9,11 @@ cd sway
 cargo install --path ./forc
 ```
 
-In the `sway/test/src/sdk-harness` directory is a bash build script. To run it:
+In the `sway/test/src/sdk-harness`, compile all the Sway programs in the workspace:
 
 ```sh
 cd sway/test/src/sdk-harness
-./build.sh
+forc build
```

Comment:
```suggestion
cargo run --bin=forc build
```

I think we might want to suggest running `forc` via `cargo run` here, in case someone using `fuelup` has a `forc` version that has a different behavior from someone working on master/a feature branch.